### ISSUE TITLE
data.csv missing

### ISSUE THREAD

**Issue by GINNOV** [raviolli77/apple_watch_data_analysis#3]
thank you for sharing this!
After processing, the data.csv file doesn't get generated. Also I noticed that in your video you have sleep data and in my case (I collect them) it doesn't show up. Any clue what could be the issue?

**Comment by raviolli77** [comment]
@GINNOV 

Yes, in the video associated I mentioned that I didn't include the entire data source! 

I haven't worked on this project for quite some time but I can ask the friend i extracted the source from if they consent to uploading their entire data source, if so I will upload the full data source.

**Comment by GINNOV** [comment]
The data isn’t the issue. As I mentioned I did collect the sleep data. It is the code that doesn’t process it
